lc56.py

```diff
@@ -1 +1,24 @@
+class Solution:
+    def merge(self, intervals: List[List[int]]) -> List[List[int]]:
+        def isOverlap(a,b):
+            maxStart = max(a[0],b[0])
+            minFinish = min(a[1],b[1])
+            return (minFinish >=  maxStart) # overlap
+        
+        intervals.sort(key = lambda x:x[0])
```

Comment:
In an interview I would check with the interviewer first to make sure it is alright to modify arguments in this way and discuss the tradeoffs.